pkg/watcher/internal/metricsprovider/signalfx.go

```diff
@@ -346,7 +369,7 @@ Sample metaData payload:
             "created": 1614534848000,
             "creator": null,
             "dimensions": {
-                "host": "test.dev.com",
+                "[hostKey]": "test.dev.com",
```

Comment:
1. I may be mistaken, but it seems to me the right key here is the value of `hostKey` which by default is `host`
2. Why are the square brackets introduced?